packages/backend-deployer/src/cdk_deployer.ts

```diff
@@ -154,6 +154,14 @@ export class CDKDeployer implements BackendDeployer {
             : HotswapMode.FULL_DEPLOYMENT,
       });
     } catch (error) {
+      await this.ioHost.notify({
+        message: 'Deployment failed',
+        code: 'DEPLOY_FAILED',
+        action: 'amplify',
+        time: new Date(),
+        level: 'debug',
```

Comment:
nit: this should be `error` in case there are consumers interested in this event in future.